src/middleware/authorisation.coffee

```diff
@@ -128,6 +136,7 @@ exports.authorise = (ctx, done) ->
     if config.authentication.enableBasicAuthentication
       ctx.set "WWW-Authenticate", "Basic"
     logger.info "The request, '" + ctx.request.path + "', is not authorised to access any channels."
+    auditing.sendAuditEvent genAuthAudit(ctx.ip), -> logger.info 'Processed nodeAuthentication audit'
```

Comment:
I've been using logger.debug for the audits, mostly because it fills up the logs with a lot of noise (e.g. auths from mediator heartbeats).

I know this is probably a low volume audit and it's probably not the same issue, but maybe for consistency we should keep them on debug?